applications/fishing-map/src/features/download/download.config.ts

```diff
@@ -0,0 +1,57 @@
+import { ChoiceOption } from '@globalfishingwatch/ui-components/dist/choice'
+import { t } from 'features/i18n/i18n'
+
+export enum Format {
```

Comment:
At some point we decided that by convention we would favor TS union string literal types (`'csv' | 'tif' | 'netcdf'`)instead of enums, but since we haven't put an eslint rule in place it's free for all.